src/prettytable/prettytable.py

```diff
@@ -183,9 +208,14 @@ def __init__(self, field_names=None, **kwargs):
         self._left_padding_width = kwargs["left_padding_width"] or None
         self._right_padding_width = kwargs["right_padding_width"] or None
 
-        self._vertical_char = kwargs["vertical_char"] or "|"
-        self._horizontal_char = kwargs["horizontal_char"] or "-"
-        self._junction_char = kwargs["junction_char"] or "+"
+        # color codes to theme color, then back to base
+        # _theme["border"] + kwargs["char"] + _theme["base"]
+        self._theme = kwargs["theme"] or THEME.DEFAULT
+        # Converted to try except switch because the "or" keyword wasn't working
```

Comment:
Is this comment still valid? I don't see a `try`/`except`

Reply:
No, not anymore. Let me change that.